app/views/clone_account_memberships/index.html.haml

```diff
@@ -0,0 +1,18 @@
+= content_for :tabnav do
+  = render partial: "admin/shared/tabnav_users", locals: {secondary_tab: "accounts"}
+= content_for :sidebar do
+  = render :partial => 'admin/shared/sidenav_users', locals: { sidenav_tab: 'users' }
+
+= content_for :h1 do
+  = current_facility
+
+%header
+  %h2= "Clone Account Memberships to #{@user.full_name}"
+
+= form_tag url_for(action: :search), id: "ajax_form", method: :get do
+  = label_tag :search_term, "Search by name or email" # t(".label.search")
```

Comment:
This one should definitely be I18n-ed because this is a shared form and I believe we override it in NU and other schools to reference netids.